api/end_device_services.proto

```diff
@@ -80,6 +80,9 @@ service EndDeviceRegistry {
     };
   };
 
+  // Update a batch of end devices, changing the fields specified by the field mask to the provided values.
+  rpc BatchUpdate(BatchUpdateEndDeviceRequest) returns (EndDevices);
```

Comment:
Do we really want to introduce an RPC that is able to update many fields of many end devices? Isn't it enough to have an RPC that only updates the LastSeenAt?

Reply:
Indeed, I've proposed this one for batch location updates and activation statuses.

Reply:
Okay, but then we can't really write a properly optimized query for it (like in https://github.com/TheThingsNetwork/lorawan-stack/pull/5290#discussion_r821447695) and we can ask ourselves what's the point of batching updates in a single RPC in the first place

Reply:
I'm ok with specializing the RPC then, and we add subsequent RPCs in the future perhaps.

Reply:
see https://github.com/TheThingsNetwork/lorawan-stack/pull/5290#discussion_r822703050